src/versions.ts

```diff
@@ -106,4 +106,13 @@ export const versions: Versions = {
     { version: "4", eol: "2012-02-29" },
     { version: "3", eol: "2010-10-31" },
   ],
+
+  // https://www.postgresql.org/support/versioning/
+  postgresql: [
+    { version: "13.x", eol: "2025-11-13" },
+    { version: "12.x", eol: "2024-11-14" },
+    { version: "11.x", eol: "2023-11-09" },
+    { version: "10.x", eol: "2022-11-10" },
+    { version: "9.x", eol: "2021-11-11" },
```

Comment:
This last one should be `9.6.x`, otherwise it would give green badge even for 9.0 which has been EOL since 2015